HomeScreen-already-sign-in.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="script.js"></script>
+    <link rel="stylesheet" type="text/css" href="./style2.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+
+  <body>
+   <div class="container">
+      <div class="searching">  
+          <div class="logo-circle">
+              <div class="logo-text"> Logo </div>
+          </div>
```

Comment:
Please minimize this into 1 class only. We dont need logo-circle and logo-text at the same time